.claude/plans/house-differential-style-v1.md

```
@@ -0,0 +1,223 @@
+recipes  = the rung-3 tactics RCR / ASC / CR (+ TR, CAS as needed)
+style    = the House program: WHICH recipes fire, in WHAT order, with WHICH
+           periphery budget — a policy, applicable at any rung
+```
+
+## 1. What is already shipped (the parts list, file-cited)
+
+| House move | Shipped surface | Where |
+|---|---|---|
+| The whiteboard: ranked hypotheses from an observation | RCR abduction over a shared predicate → `Frontier { candidates, gaps }`, `Candidate { stmt, truth, premises, rung, tactic }`, `ReasoningGap` | planner `nars/tactics.rs:177 rcr_abduce`, `:71 Candidate`, `:88 ReasoningGap` |
+| Attack the leading hypothesis | ASC: `challenge_target = ⟨1−f, c⟩`; counter-evidence admitted ONLY with a disjoint stamp; self-refutation `BlockedSelfReference` | `tactics.rs:457 challenge_target`, `:479 asc_challenge`, `:463 AscOutcome` |
+| Thesis vs antithesis on one claim | CR: NARS revision, `|f₁−f₂|` contradiction depth preserved, never averaged | `tactics.rs:503 cr_synthesize` |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)